src/api/SessionAPI.ts

```diff
@@ -0,0 +1,14 @@
+import * as APIUtils from "./APIUtils";
+import { Session } from "../types";
+
+export type ClassIndex = {
+  id: number;
+  name: string;
+  facilitator_id: number;
+};
+
+const getSessions = (): Promise<Session[]> => APIUtils.get("/sessions");
```

Comment:
I believe the get /sessions endpoint only returns the id, season, and year ([serializer](https://github.com/uwblueprint/project-read-backend/blob/06e7ca3be7712a632c54d1e8c16b009083fff378/enrolments/serializers.py#L8)) so this can lead to some funky behaviour if we reference the other fields on the `Session` type

we can define a `SessionListResponse` type (similar to [this](https://github.com/uwblueprint/project-read-frontend/blob/ca6fb7b49684339d0322337eca8a327b374cc807/src/api/FamilyAPI.ts#L21)) that references the `Session` type!